source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj

```diff
@@ -5,7 +5,7 @@
     <Authors>Microsoft</Authors>
     <Summary>Adaptive Card Template for WinRT</Summary>
     <Description>This library implements classes for building and serializing adaptive card objects</Description>
-    <PackageId>AdaptiveCards.Template</PackageId>
```

Comment:
Will this introduce a breaking change or a nuget package name?

Reply:
yes, it's requested by Widget team